src/dc_tools.rs

```diff
@@ -450,21 +450,25 @@ pub fn dc_open_file_std<P: AsRef<std::path::Path>>(
     }
 }
 
-pub(crate) async fn dc_get_next_backup_path(
+pub(crate) async fn dc_get_next_backup_path_new(
     folder: impl AsRef<Path>,
     backup_time: i64,
-) -> Result<PathBuf, Error> {
+) -> Result<(PathBuf, PathBuf), Error> {
```

Comment:
Can you add some documentation to what the two different `PathBuf`s are? Might be best to return a struct even